src/PostCampaign.php

```diff
@@ -437,28 +434,25 @@ private static function get_theme_defaults( $theme_json ) {
 	public static function css_vars( array $meta ): array {
 		$theme = self::get_theme( $meta );
 
-		$is_new_theme = ! in_array( $theme, self::LEGACY_THEMES, true );
-		if ( $is_new_theme ) {
-			$potential_new_version = str_replace( '-new', '', $theme );
+		// As a way to make new themes use the same config file, check if removing "-new" gives a legacy theme name.
```

Comment:
Only removed this check as it will now always be true. https://github.com/greenpeace/planet4-master-theme/pull/1444/files?diff=split&w=1#diff-e75d22944aee56e2f11a3fe7d86bc1f7acec3be0a6a92d3cbeb45c871558d324L441